include/klee/util/Bits.h

```diff
@@ -88,9 +95,11 @@ namespace klee {
     // @pre((x&(x-1)) == 0)
     // @post((1 << retval) == x)
     inline unsigned indexOfSingleBit(uint64_t x) {
+      assert((x & (x - 1)) == 0);
       unsigned res = bits32::indexOfSingleBit((unsigned) (x | (x>>32)));
-      if (x&((uint64_t) 0xFFFFFFFF << 32))
-	  res += 32;
+      if (x & ((uint64_t)0xFFFFFFFF << 32))
```

Comment:
I wasn't complaining about the formatting I was suggesting that we do the casting in a more readable and less ambiguous way by using the `UINT64_C()` macro.